openforcefield/utils/toolkits.py

```diff
@@ -35,6 +35,7 @@
     "UndefinedStereochemistryError",
     "GAFFAtomTypeWarning",
     "ToolkitWrapper",
+    "BuiltInToolkitWrapper",
```

Comment:
@j-wags Is this an old piece that should be removed?

Reply:
That might look out of place, but it should stay (I don't think `__all__` got updated when that class was added. I do `from openforcefield.utils.toolkits import *` a lot, so that's a minor point of frustration)